schema/sale/sorts.js

```diff
@@ -40,6 +40,12 @@ export default {
       ELIGIBLE_SALE_ARTWORKS_COUNT_DESC: {
         value: '-eligible_sale_artworks_count',
       },
+      TIMELY_AT_NAME_ASC: {
```

Comment:
I actually wasn't sure where these were used-- are they just convenience methods for consumers of metaphysics?

It seemed like all of the other sorts were here so I thought it best to add these as well, but /shrug